pipenv/core.py

```diff
@@ -2194,6 +2194,19 @@ def do_graph(bare=False, json=False, reverse=False):
     if reverse:
         flag = '--reverse'
 
+    if not project.virtualenv_exists:
+        click.echo(
+            u'{0}: No virtualenv has been created for this project yet! Consider '
+            u'running {1} first to automatically generate a one for you or see'
```

Comment:
I think we need to remove "a" from "generate a one for you". We could probably shorten it to just "generate one".